emoji/src/EmojiRow/EmojiRow.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import style from './style.module.css';
+
+export default function EmojiRow (props) {
+    const codePointHex = props.symbol.codePointAt(0).toString(16);
```

Comment:
Always try to check if prop exist. It will help you prevent unexpected errors.
```
const codePointHex =symbol && symbol.codePointAt(0).toString(16)
```